src/poetry/utils/helpers.py

```diff
@@ -133,6 +138,11 @@ def download_file(
                             last_percent = percent
                             update_context(f"Downloading {url} {percent:3}%")
 
+def download_file_with_curl(
+    url: str,
+    dest: str,
+) -> None:
+    subprocess.run(['curl', url, '--silent', '--output', dest], check=True)
```

Comment:
can you add a docstring explaining that this is significantly faster than the Python-native solution?

Reply:
:+1: